packages/field-base/src/field-mixin.d.ts

```diff
@@ -33,23 +33,25 @@ export declare class FieldMixinClass {
   helperText: string | null | undefined;
 
   /**
-   * Error to show when the field is invalid.
+   * Error message to show when the field is invalid.
    *
    * @attr {string} error-message
    */
-  errorMessage: string;
-
-  protected readonly _ariaAttr: 'aria-labelledby' | 'aria-describedby';
-
-  protected _ariaTargetChanged(target: HTMLElement): void;
+  errorMessage: string | null | undefined;
 
   protected readonly _errorNode: HTMLElement;
 
   protected readonly _helperNode?: HTMLElement;
 
   protected _helperTextChanged(helperText: string | null | undefined): void;
 
-  protected _updateAriaAttribute(target: HTMLElement, invalid: boolean, helperId: string): void;
+  protected _ariaTargetChanged(target: HTMLElement): void;
+
+  protected _updateErrorMessage(invalid: boolean, errorMessage: string | null | undefined): void;
+
+  protected _requiredChanged(required: boolean): void;
```

Comment:
Should this callback be moved rather in `ValidateMixin`?

Reply:
I think it's fine to keep the observer in scope of `FieldMixin` as there is no use for it in `ValidateMixin`.